proposals/json-td/index.html

```diff
@@ -51,7 +58,8 @@
     </section>
     <section>
       <h2>Introduction</h2>
-      <p>The goal of the Web of Things is to create a decentralized Internet of Things by giving connected devices URLs on the web to make them linkable and discoverable, and defining a standard data model and APIs to make them interoperable.</p>
+      <p>The Web of Things is a refinement of the IoT by integrating things not only to the Internet (network), but to the Web Architecture (application). The goal of the Web of Things is to create a decentralized Internet of Things by giving connected devices URLs on the web to make them linkable and discoverable, and defining a standard data model and APIs to make them interoperable.</p>
```

Comment:
Why is Web Architecture capitalised? I'm not really sure what that term is referring to.

How about "The Web of Things is intended as a unifying application layer for the Internet of Things."?